nemo_text_processing/text_normalization/normalize.py

```diff
@@ -22,10 +22,11 @@
 from nemo_text_processing.text_normalization.verbalizers.verbalize_final import VerbalizeFinalFst
 from tqdm import tqdm
 
+INPUT_CASE = "lower_cased"  # cased
```

Comment:
can we infer this from the input itself?  and in the case when it's not possible, use this param? could you add it to the arguments?

Reply:
let me figure out how to add it to args